app/views/directives/osc-secrets.html

```diff
@@ -0,0 +1,62 @@
+<ng-form name="secretsForm" class="osc-secrets-form">
+  <div ng-repeat="pickedSecret in pickedSecrets">
+    <div class="form-group">
+      <div class="row picked-secret">
+        <div ng-class="type === 'Source Secret' ? 'col-sm-6' : 'col-sm-12'">
+          <label class="picker-label" ng-show="$index === 0">{{type | startCase}}</label>
+          <ui-select ng-model="pickedSecret.name">
+            <ui-select-match allow-clear="{{type === 'Source Secret' ? false : true}}" placeholder="Secret name">{{$select.selected}}</ui-select-match>
```

Comment:
@jwforres and I looked at it, and the `allow-clear` interaction with ui-select seems really strange. I'd rather just have an empty option in the select that I can pick.